docs/dev/getting-set-up.md

```diff
@@ -6,8 +6,11 @@ This section of the docs covers our practices for working with `git` on our code
 
 For a more complete git tutorials we recommend checking out:
 
-- [Atlassian's git tutorial](https://www.atlassian.com/git/tutorials) -- Beginner friendly introductions to the git command line interface
-- [Setting up git for GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git) -- Configuring git to work with your GitHub user account
+[Atlassian's git tutorial](https://www.atlassian.com/git/tutorials)
+: Beginner friendly introductions to the git command line interface
+
+[Setting up git for GitHub](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git)
+: Configuring git to work with your GitHub user account
```

Comment:
Do you want to change the links to be the other format?

Reply:
Since these are alone on their lines, that’s not necessary. I do it to make body text easier to read, not to be complicated haha.

Reply:
Gotcha, wasn't totally sure but that makes sense.  Will be attentive going forward to that.